engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-action-packages.controller.js

```diff
@@ -21,17 +21,23 @@ angular.module('Bastion.content-hosts').controller('ContentHostsBulkActionPackag
 
         function successMessage(type) {
             var messages = {
-                install: translate("Succesfully scheduled package installation"),
-                update: translate("Succesfully scheduled package update"),
-                remove: translate("Succesfully scheduled package removal")
+                install: translate("Successfully scheduled package installation"),
+                update: translate("Successfully scheduled package update"),
+                remove: translate("Successfully scheduled package removal"),
+                "update all": translate("Successfully scheduled an update of all packages")
             };
             return messages[type];
         }
 
         function installParams() {
             var params = $scope.nutupane.getAllSelectedResults();
             params['content_type'] = $scope.content.contentType;
-            params.content = $scope.content.content.split(/ *, */);
+            params.content = null;
+            if ($scope.content.action !== "update all") {
+                params.content = $scope.content.content.split(/ *, */);
+            } else {
+                params['update_all'] = true;
+            }
```

Comment:
How would you feel about reversing the logic for this so it's positive?

Something like this (untested):

```javascript
params.content = $scope.content.content.split(/ *, */);

if ($scope.content.action == "update all") {
    params['update_all'] = true;
    params.content = null;
}
```